### PR TITLE
Doc/update readme

### PR DESCRIPTION
**Pull request by applovinAsh**
https://app.asana.com/0/1200664767648537/1205892373505164

### CODE REVIEW

**Review by alhiwatan — APPROVED** [review]
It looks good to me.

File: README.md [hunk 1]
```diff
@@ -7,5 +7,75 @@ Check out our integration docs [here](https://dash.applovin.com/documentation/me
...
+    super.configureFlutterEngine(flutterEngine)
+    // Your Kotlin code here...
+}
+```
+
+### iOS 
+#### 1. Adding adapters to your podfile:
+- Locate your `Podfile` in the `/ios` folder.
+- Open the Podfile with a text editor or IDE.
+- Add the adapter pods to your application as specified in the documentation. It will look something like this
```
Comment (Lorenzo45) [inline README.md:62]: Would be nice to link to the docs here as well

File: README.md [hunk 2]
```diff
@@ -7,5 +7,75 @@ Check out our integration docs [here](https://dash.applovin.com/documentation/me
...
+target 'Runner' do
+  flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))
+  pod `AppLovinSDK`
+end
+```
+#### 2. Install the Pods:
+- Save the `Podfile`.
+- Open a terminal window, navigate to the root of your Flutter project, and run the following command to install the pods:
+```flutter pub get```
+- Optionally, you may also run `pod install` within the `ios` directory.
```
Comment (Lorenzo45) [inline README.md:73]: Is `pod install` an alternative to `flutter pub get`? If so, would be slightly less confusing to update the wording to "Alternatively,"

**Comment by applovinAsh** [comment]
Changes made

File: README.md [hunk 3]
```diff
@@ -7,5 +7,75 @@ Check out our integration docs [here](https://dash.applovin.com/documentation/me
 ## Downloads
 See [pub.dev](https://pub.dev/packages/applovin_max) for the latest releases of the plugin.
 
+## Demo App Instructions
+To get started with the demo app, please ensure Flutter is installed on your system. Once everything is properly installed, follow the instructions below to get the demo application up and running. 
+
+1. Update the `SDK_KEY` and ad unit IDs in the `main.dart` file [here](https://github.com/AppLovin/AppLovin-MAX-Flutter/blob/73c5a281439791f4b81f595ab795f6b99cbc9457/applovin_max/example/lib/main.dart#L28).
```
Comment (Lorenzo45) [inline README.md:13]: We should specify where to "get" the SDK key, not "set" it. For SDK key, easiest way would probably be to link to the docs here:
https://dash.applovin.com/documentation/mediation/flutter/getting-started/integration#initialize-the-sdk

For ad unit IDs, we can link to this: https://dash.applovin.com/o/mediation/ad_units

Comment (applovinAsh) [reply README.md]: My bad. updated.

**Comment by applovinAsh** [comment]
Changes made

File: README.md [hunk 4]
```diff
@@ -7,5 +7,77 @@ Check out our integration docs [here](https://dash.applovin.com/documentation/me
 ## Downloads
 See [pub.dev](https://pub.dev/packages/applovin_max) for the latest releases of the plugin.
 
+## Demo App Instructions
+To get started with the demo app, please ensure Flutter is installed on your system. Once everything is properly installed, follow the instructions below to get the demo application up and running. 
+
+1. Obtain your AppLovin SDK Key from the dashboard [here](https://dash.applovin.com/documentation/mediation/flutter/getting-started/integration#initialize-the-sdk). 
+2. Obtain your Ad Unit IDs from the dashboard [here](https://dash.applovin.com/o/mediation/ad_units).
+3. Update the `SDK_KEY` and Ad Unit IDs in the `main.dart` file .
```
Comment (Lorenzo45) [inline README.md:15]: nit: remove extra space
```suggestion
3. Update the `SDK_KEY` and Ad Unit IDs in the `main.dart` file. 
```

**Review by Lorenzo45 — APPROVED** [review]
Good to go!

**Pull request merged**